handlers/token_balance_api.py

```diff
@@ -0,0 +1,63 @@
+import json
+from typing import Optional
+
+from aws_lambda_context import LambdaContext
+
+from common.errors import ApiError
+from usecases.token_balances_api import TokenBalancesApi
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def get_token_balances(
+    event: ApiGatewayEvent,
+    _context: LambdaContext,
+    token_balances_api: Optional[TokenBalancesApi] = None
+) -> dict:
+    """Get token balances from user search"""
+
+    token_balances_api = token_balances_api or TokenBalancesApi()
```

Comment:
This is equivalent to setting a default in the function parameters, right?

In case you think it's better:
```py
def get_token_balances(
    event: ApiGatewayEvent,
    _context: LambdaContext,
    token_balances_api: TokenBalancesApi = TokenBalancesApi()
) -> dict:
```

Reply:
I like your suggestion, refactored in 5ec74623639c35737c55020a1ac1e892927b7c53